src/reactviews/pages/SchemaDesigner/toolbar/publishChangesDialogButton.tsx

```diff
@@ -79,10 +78,11 @@ export function PublishChangesDialogButton() {
      */
     const triggerButton = () => {
         return (
-            <ToolbarButton
+            <Button
```

Comment:
Converting toolbar buttons to button to make them small. Haven't noticed any degradation of keyboard accessibility.